Automattic/wp-calypso
# Add back to intro button

Related to https://github.com/Automattic/dotcom-forge/issues/2940

## Proposed Changes
Add a back to `intro` button on the last step of the flow

<img width="1003" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/2a3b1b8f-faae-4622-9804-4560787ff387">


## Testing Instructions
Pull and run this branch or use calypso live link
Navigate to `setup/domain-transfer/complete` and check if the `Transfer more domains` is visible
Clicking on it, it should redirect you to `setup/domain-transfer/intro`

[More context](https://github.com/Automattic/dotcom-forge/issues/2940)
